pluma/pluma-commands-search.c

```diff
@@ -144,6 +144,11 @@ text_not_found (PlumaWindow *window,
 	g_free (searched);
 }
 
+/* This is the string with last replace statement. The reason to use some kind of
+   global variable is to keep existing methods' signatures not changed (to minimize
```

Comment:
This PR already has changes in 3 functions 🙂

Reply:
@monsta Three is less than six. Of course I can send this argument through all  sequence of calls if this global variable looks ugly. Or remove this comment.